kwanghyun/controllers/likeControllers.js

```diff
@@ -0,0 +1,21 @@
+const likeService = require("../services/likeService");
+
+const likes = async (req, res) => {
+  try {
+    const { userId, postId } = req.body;
+
+    if (!userId || !postId) {
+      return res.status(400).json({ message: "KEY_ERROR" });
+    }
+
+    await likeService.likes(userId, postId);
+    res.status(200).json({ message: "LIKE_CHECK" });
```

Comment:
```suggestion
    res.status(201).json({ message: "LIKE_CHECK" });
```
데이터를 생성 및 수정하는 것을 의미하는 POST method를 활용하고 있으면서 동시에 실질적으로 api 내부 로직 또한 좋아요라는 데이터를 생성하고 있으므로 그에 적합한 status code를 사용하는 것이 옳습니다.